app/components/views/Home/index.tsx

```diff
@@ -121,17 +122,37 @@ const useProvider = (): [
   ];
 };
 
+const useIsMobile = (): { isMobile: boolean } => {
```

Comment:
Can we just use CSS breakpoints to show/hide the `.mobileMenu` class?

Reply:
no prob